modules/andes-core/client/src/test/java/org/wso2/andes/unit/message/TestAMQSession.java

```diff
@@ -64,6 +64,10 @@ public void sendCommit() throws AMQException, FailoverException
 
     }
 
+    public void sendReject(long deliveryTag, boolean reQueue) {
```

Comment:
This is a test method used to mock some functionality. Only needed methods are implemented.